nvdaHelper/localWin10/uwpOcr.h

```diff
@@ -0,0 +1,30 @@
+/*
+Header for C dll bridge to UWP OCR.
+This file is a part of the NVDA project.
+URL: http://www.nvaccess.org/
+Copyright 2017 NV Access Limited.
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License version 2.0, as published by
+    the Free Software Foundation.
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+This license can be found at:
+http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+*/
+
+#pragma once
+#define export __declspec(dllexport) 

```

Comment:
> In case I dont notice later, how does this become an import?

This exports the symbol from the dll so that a caller loading the dll can access it. We then access it using ctypes (which internally uses [GetProcAddress)](https://msdn.microsoft.com/en-us/library/windows/desktop/ms683212(v=vs.85).aspx).